diff --git a/node_modules/@automattic/generate-password/dist/cjs/index.js b/node_modules/@automattic/generate-password/dist/cjs/index.js
index 05d6c13..bfd7f48 100644
--- a/node_modules/@automattic/generate-password/dist/cjs/index.js
+++ b/node_modules/@automattic/generate-password/dist/cjs/index.js
@@ -30,7 +30,7 @@ function generatePassword({ length = 24, useNumbers = true, useSpecialChars = tr
     // @link https://dimitri.xyz/random-ints-from-random-bits/
     for (let i = 0; i < length; i++) {
         do {
-            window.crypto.getRandomValues(randomNumber);
+            globalThis.crypto.getRandomValues(randomNumber);
         } while (randomNumber[0] >= characterPool.length);
         password += characterPool[randomNumber[0]];
     }
diff --git a/node_modules/@automattic/generate-password/dist/esm/index.js b/node_modules/@automattic/generate-password/dist/esm/index.js
index e2d5910..6013fbd 100644
--- a/node_modules/@automattic/generate-password/dist/esm/index.js
+++ b/node_modules/@automattic/generate-password/dist/esm/index.js
@@ -27,7 +27,7 @@ export function generatePassword({ length = 24, useNumbers = true, useSpecialCha
     // @link https://dimitri.xyz/random-ints-from-random-bits/
     for (let i = 0; i < length; i++) {
         do {
-            window.crypto.getRandomValues(randomNumber);
+            globalThis.crypto.getRandomValues(randomNumber);
         } while (randomNumber[0] >= characterPool.length);
         password += characterPool[randomNumber[0]];
     }
diff --git a/node_modules/@automattic/generate-password/src/index.ts b/node_modules/@automattic/generate-password/src/index.ts
index b1a7d7f..51469c7 100644
--- a/node_modules/@automattic/generate-password/src/index.ts
+++ b/node_modules/@automattic/generate-password/src/index.ts
@@ -57,7 +57,7 @@ export function generatePassword( {
 	// @link https://dimitri.xyz/random-ints-from-random-bits/
 	for ( let i = 0; i < length; i++ ) {
 		do {
-			window.crypto.getRandomValues( randomNumber );
+			globalThis.crypto.getRandomValues( randomNumber );
 		} while ( randomNumber[ 0 ] >= characterPool.length );
 
 		password += characterPool[ randomNumber[ 0 ] ];
