diff --git a/node_modules/@types/archiver/index.d.ts b/node_modules/@types/archiver/index.d.ts
index 03d2d7d..88abe66 100644
--- a/node_modules/@types/archiver/index.d.ts
+++ b/node_modules/@types/archiver/index.d.ts
@@ -106,6 +106,7 @@ declare namespace archiver {
 
     interface CoreOptions {
         statConcurrency?: number | undefined;
+        followSymlinks?: boolean;
     }
 
     interface TransformOptions {
