diff --git a/node_modules/archiver/lib/core.js b/node_modules/archiver/lib/core.js
index 7c0a74d..2934fe5 100644
--- a/node_modules/archiver/lib/core.js
+++ b/node_modules/archiver/lib/core.js
@@ -631,7 +631,8 @@ Archiver.prototype.directory = function(dirpath, destpath, data) {
 
   var globOptions = {
     stat: true,
-    dot: true
+    dot: true,
+    follow: this.options.followSymlinks
   };
 
   function onGlobEnd() {
