diff --git a/node_modules/archiver/lib/core.js b/node_modules/archiver/lib/core.js
index 7c0a74d..c70117b 100644
--- a/node_modules/archiver/lib/core.js
+++ b/node_modules/archiver/lib/core.js
@@ -631,7 +631,8 @@ Archiver.prototype.directory = function(dirpath, destpath, data) {
 
   var globOptions = {
     stat: true,
-    dot: true
+    dot: true,
+    follow: this.options.followSymlinks
   };
 
   function onGlobEnd() {
@@ -922,6 +923,7 @@ module.exports = Archiver;
  * @global
  * @property {Number} [statConcurrency=4] Sets the number of workers used to
  * process the internal fs stat queue.
+ * @property {Boolean} [followSymLinks=false] Sets whether to follow symlinks.
  */
 
 /**
