diff --git a/node_modules/playwright-core/lib/server/browserContext.js b/node_modules/playwright-core/lib/server/browserContext.js
index f7a5ef2..87a2764 100644
--- a/node_modules/playwright-core/lib/server/browserContext.js
+++ b/node_modules/playwright-core/lib/server/browserContext.js
@@ -181,12 +181,14 @@ if (navigator.serviceWorker) navigator.serviceWorker.register = async () => { co
     await this.setStorageState(progress, this._options.storageState, "resetForReuse");
     await page?.resetForReuse(progress);
   }
-  _browserClosed() {
+	_browserClosed() {
+	  console.trace('BrowserContext::_browserClosed')
     for (const page of this.pages())
       page._didClose();
     this._didCloseInternal();
   }
-  _didCloseInternal() {
+	_didCloseInternal() {
+  console.trace('BrowserContext::_didCloseInternal')
     if (this._closedStatus === "closed") {
       return;
     }
@@ -394,14 +396,16 @@ if (navigator.serviceWorker) navigator.serviceWorker.register = async () => { co
   async _deleteAllDownloads() {
     await Promise.all(Array.from(this._downloads).map((download) => download.artifact.deleteOnContextClose()));
   }
-  async _deleteAllTempDirs() {
+	async _deleteAllTempDirs() {
+	  console.log('BrowserContext::_deleteAllTempDirs', this._tempDirs);
     await Promise.all(this._tempDirs.map(async (dir) => await import_fs.default.promises.unlink(dir).catch((e) => {
     })));
   }
   setCustomCloseHandler(handler) {
     this._customCloseHandler = handler;
   }
-  async close(options) {
+	async close(options) {
+	  console.log('BrowserContext::close start', this._closedStatus);
     if (this._closedStatus === "open") {
       if (options.reason)
         this._closeReason = options.reason;
@@ -415,13 +419,17 @@ if (navigator.serviceWorker) navigator.serviceWorker.register = async () => { co
         if (context === this)
           promises.push(artifact.finishedPromise());
       }
+      console.log('BrowserContext::close _customCloseHandler start');
       if (this._customCloseHandler) {
         await this._customCloseHandler();
       } else {
         await this.doClose(options.reason);
       }
+      console.log('BrowserContext::close _customCloseHandler resolved');
       promises.push(this._deleteAllDownloads());
+      console.log('BrowserContext::close deleted all downloads');
       promises.push(this._deleteAllTempDirs());
+      console.log('BrowserContext::close deleted all temp dirs');
       await Promise.all(promises);
       if (!this._customCloseHandler)
         this._didCloseInternal();
diff --git a/node_modules/playwright-core/lib/server/electron/electron.js b/node_modules/playwright-core/lib/server/electron/electron.js
index 3fb4963..6cd3d49 100644
--- a/node_modules/playwright-core/lib/server/electron/electron.js
+++ b/node_modules/playwright-core/lib/server/electron/electron.js
@@ -75,13 +75,17 @@ class ElectronApplication extends import_instrumentation.SdkObject {
     });
     this._nodeSession.on("Runtime.consoleAPICalled", (event) => this._onConsoleAPI(event));
     const appClosePromise = new Promise((f) => this.once(ElectronApplication.Events.Close, f));
-    this._browserContext.setCustomCloseHandler(async () => {
+	  this._browserContext.setCustomCloseHandler(async () => {
+		console.log('ElectronApplication customCloseHandler start');
       await this._browserContext.stopVideoRecording();
       const electronHandle = await this._nodeElectronHandlePromise;
+      console.log('ElectronApplication customCloseHandler got electron handle');
       await electronHandle.evaluate(({ app }) => app.quit()).catch(() => {
       });
+      console.log('ElectronApplication customCloseHandler evaluated `app.quit`');
       this._nodeConnection.close();
       await appClosePromise;
+      console.log('ElectronApplication customCloseHandler finished');
     });
   }
   static {
@@ -156,11 +160,6 @@ class Electron extends import_instrumentation.SdkObject {
       electronArguments.unshift("-r", require.resolve("./loader"));
     }
     let shell = false;
-    if (process.platform === "win32") {
-      shell = true;
-      command = `"${command}"`;
-      electronArguments = electronArguments.map((arg) => `"${arg}"`);
-    }
     delete env.NODE_OPTIONS;
     const { launchedProcess, gracefullyClose, kill } = await (0, import_processLauncher.launchProcess)({
       command,
diff --git a/node_modules/playwright-core/lib/server/utils/processLauncher.js b/node_modules/playwright-core/lib/server/utils/processLauncher.js
index 954d5e6..df35158 100644
--- a/node_modules/playwright-core/lib/server/utils/processLauncher.js
+++ b/node_modules/playwright-core/lib/server/utils/processLauncher.js
@@ -149,7 +149,11 @@ async function launchProcess(options) {
   let processClosed = false;
   let fulfillCleanup = () => {
   };
-  const waitForCleanup = new Promise((f) => fulfillCleanup = f);
+	const waitForCleanup = new Promise((f) => fulfillCleanup = f);
+ spawnedProcess.once("exit", (exitCode, signal) => {
+    options.log(`[pid=${spawnedProcess.pid}] <process exit event: exitCode=${exitCode}, signal=${signal}>`);
+  });
+
   spawnedProcess.once("close", (exitCode, signal) => {
     options.log(`[pid=${spawnedProcess.pid}] <process did exit: exitCode=${exitCode}, signal=${signal}>`);
     processClosed = true;
