diff --git a/node_modules/@types/archiver/index.d.ts b/node_modules/@types/archiver/index.d.ts
index 03d2d7d..76ad877 100644
--- a/node_modules/@types/archiver/index.d.ts
+++ b/node_modules/@types/archiver/index.d.ts
@@ -106,6 +106,7 @@
 
     interface CoreOptions {
         statConcurrency?: number | undefined;
+        followSymlinks?: boolean | undefined;
     }
 
     interface TransformOptions {
