diff --git a/node_modules/@wp-playground/wordpress/index.cjs b/node_modules/@wp-playground/wordpress/index.cjs
index 0f93ab4..97a2ad1 100644
--- a/node_modules/@wp-playground/wordpress/index.cjs
+++ b/node_modules/@wp-playground/wordpress/index.cjs
@@ -469,7 +469,7 @@ class WP_Config_Transformer {
 			&& ( T_WHITESPACE === $token[0] || T_COMMENT === $token[0] || T_DOC_COMMENT === $token[0] );
 	}
 }
-`;async function g(t,e){const n=r.joinPaths(e,"wp-config.php"),i={DB_NAME:"wordpress"};if(!t.fileExists(n)&&t.fileExists(r.joinPaths(e,"wp-config-sample.php"))&&await t.writeFile(n,await t.readFileAsBuffer(r.joinPaths(e,"wp-config-sample.php"))),!t.fileExists(n))return;const a=r.phpVars({wpConfigPath:n,constants:i});if((await t.run({code:`${$}
+`;async function g(t,e){const n=r.joinPaths(e,"wp-config.php"),i={};if(!t.fileExists(n)&&t.fileExists(r.joinPaths(e,"wp-config-sample.php"))&&await t.writeFile(n,await t.readFileAsBuffer(r.joinPaths(e,"wp-config-sample.php"))),!t.fileExists(n))return;const a=r.phpVars({wpConfigPath:n,constants:i});if((await t.run({code:`${$}
 		$wp_config_path = ${a.wpConfigPath};
 		$transformer    = WP_Config_Transformer::from_file($wp_config_path);
 		foreach ( ${a.constants} as $name => $value ) {
diff --git a/node_modules/@wp-playground/wordpress/index.js b/node_modules/@wp-playground/wordpress/index.js
index 7ef25c4..46a3b7c 100644
--- a/node_modules/@wp-playground/wordpress/index.js
+++ b/node_modules/@wp-playground/wordpress/index.js
@@ -475,9 +475,7 @@ class WP_Config_Transformer {
 }
 `;
 async function I(t, e) {
-  const n = s(e, "wp-config.php"), i = {
-    DB_NAME: "wordpress"
-  };
+  const n = s(e, "wp-config.php"), i = {};
   if (!t.fileExists(n) && t.fileExists(s(e, "wp-config-sample.php")) && await t.writeFile(
     n,
     await t.readFileAsBuffer(
