diff --git a/node_modules/yargs/lib/platform-shims/esm.mjs b/node_modules/yargs/lib/platform-shims/esm.mjs
index 1b8efe8..0e34ec4 100644
--- a/node_modules/yargs/lib/platform-shims/esm.mjs
+++ b/node_modules/yargs/lib/platform-shims/esm.mjs
@@ -61,7 +61,7 @@ export default {
   },
   stringWidth,
   y18n: y18n({
-    directory: resolve(__dirname, '../../../locales'),
+    directory: resolve(dirname(require.resolve('yargs')), 'locales'),
     updateFiles: false
   })
 }
