diff --git a/node_modules/@wordpress/ui/build-module/icon-button/icon-button.mjs b/node_modules/@wordpress/ui/build-module/icon-button/icon-button.mjs
index 1623748..d78b6c1 100644
--- a/node_modules/@wordpress/ui/build-module/icon-button/icon-button.mjs
+++ b/node_modules/@wordpress/ui/build-module/icon-button/icon-button.mjs
@@ -110,7 +110,7 @@ var IconButton = forwardRef(
     ...restProps
   }, ref) {
     const classes = clsx(style_default["icon-button"], className);
-    return /* @__PURE__ */ jsx(Tooltip.Provider, { delay: 0, children: /* @__PURE__ */ jsxs(Tooltip.Root, { children: [
+    return /* @__PURE__ */ jsx(Tooltip.Provider, { children: /* @__PURE__ */ jsxs(Tooltip.Root, { children: [
       /* @__PURE__ */ jsx(
         Tooltip.Trigger,
         {
diff --git a/node_modules/@wordpress/ui/build/icon-button/icon-button.cjs b/node_modules/@wordpress/ui/build/icon-button/icon-button.cjs
index 3f2a8a4..a5cba59 100644
--- a/node_modules/@wordpress/ui/build/icon-button/icon-button.cjs
+++ b/node_modules/@wordpress/ui/build/icon-button/icon-button.cjs
@@ -144,7 +144,7 @@ var IconButton = (0, import_element.forwardRef)(
     ...restProps
   }, ref) {
     const classes = (0, import_clsx.default)(style_default["icon-button"], className);
-    return /* @__PURE__ */ (0, import_jsx_runtime.jsx)(Tooltip.Provider, { delay: 0, children: /* @__PURE__ */ (0, import_jsx_runtime.jsxs)(Tooltip.Root, { children: [
+    return /* @__PURE__ */ (0, import_jsx_runtime.jsx)(Tooltip.Provider, { children: /* @__PURE__ */ (0, import_jsx_runtime.jsxs)(Tooltip.Root, { children: [
       /* @__PURE__ */ (0, import_jsx_runtime.jsx)(
         Tooltip.Trigger,
         {
