diff --git a/node_modules/@php-wasm/node/index.cjs b/node_modules/@php-wasm/node/index.cjs
index 1c7ca26..6158442 100644
--- a/node_modules/@php-wasm/node/index.cjs
+++ b/node_modules/@php-wasm/node/index.cjs
@@ -461083,13 +461083,6 @@ async function withXdebug(version = import_universal3.LatestSupportedPHPVersion,
           ].join("\n")
         );
       }
-      phpRuntime.FS.mkdirTree(process.cwd());
-      phpRuntime.FS.mount(
-        phpRuntime.FS.filesystems["NODEFS"],
-        { root: process.cwd() },
-        process.cwd()
-      );
-      phpRuntime.FS.chdir(process.cwd());
     }
   };
 }
diff --git a/node_modules/@php-wasm/node/index.js b/node_modules/@php-wasm/node/index.js
index df415c6..4a1b123 100644
--- a/node_modules/@php-wasm/node/index.js
+++ b/node_modules/@php-wasm/node/index.js
@@ -461058,13 +461058,6 @@ async function withXdebug(version = LatestSupportedPHPVersion3, options, xdebugO
           ].join("\n")
         );
       }
-      phpRuntime.FS.mkdirTree(process.cwd());
-      phpRuntime.FS.mount(
-        phpRuntime.FS.filesystems["NODEFS"],
-        { root: process.cwd() },
-        process.cwd()
-      );
-      phpRuntime.FS.chdir(process.cwd());
     }
   };
 }
