diff --git a/node_modules/pm2/types/index.d.ts b/node_modules/pm2/types/index.d.ts
index 77280f5..a9ef7e7 100644
--- a/node_modules/pm2/types/index.d.ts
+++ b/node_modules/pm2/types/index.d.ts
@@ -26,21 +26,21 @@ export function connect(noDaemonMode:boolean, errback: ErrCallback): void;
  * @param errback - An errback called when the script has been started.
  * The proc parameter will be a pm2 process object.
  */
-export function start(options: StartOptions, errback: ErrProcCallback): void;
+export function start(options: StartOptions, errback: ErrProcsCallback): void;
 /**
  * Starts a script that will be managed by pm2.
  * @param jsonConfigFile - The path to a JSON file that can contain the same options as the options parameter.
  * @param errback - An errback called when the script has been started.
  * The proc parameter will be a pm2 process object.
  */
-export function start(jsonConfigFile: string, errback: ErrProcCallback): void;
+export function start(jsonConfigFile: string, errback: ErrProcsCallback): void;
 /**
  * Starts a script that will be managed by pm2.
  * @param script - The path of the script to run.
  * @param errback - An errback called when the script has been started.
  * The proc parameter will be a pm2 process object.
  */
-export function start(script: string , errback: ErrProcCallback): void;
+export function start(script: string , errback: ErrProcsCallback): void;
 /**
  * Starts a script that will be managed by pm2.
  * @param script - The path of the script to run.
@@ -48,7 +48,7 @@ export function start(script: string , errback: ErrProcCallback): void;
  * @param errback - An errback called when the script has been started.
  * The proc parameter will be a pm2 process object.
  */
-export function start(script: string, options: StartOptions, errback: ErrProcCallback): void;
+export function start(script: string, options: StartOptions, errback: ErrProcsCallback): void;
 /**
  * Starts a script that will be managed by pm2.
  * @param script - The path of the script to run.
@@ -56,7 +56,7 @@ export function start(script: string, options: StartOptions, errback: ErrProcCal
  * @param errback - An errback called when the script has been started.
  * The proc parameter will be a pm2 process object.
  */
-export function start(script: string, jsonConfigFile: string, errback: ErrProcCallback): void;
+export function start(script: string, jsonConfigFile: string, errback: ErrProcsCallback): void;
 
 /**
  * Disconnects from the pm2 daemon.
@@ -715,8 +715,9 @@ type ProcessStatus = 'online' | 'stopping' | 'stopped' | 'launching' | 'errored'
 type Platform = 'ubuntu' | 'centos' | 'redhat' | 'gentoo' | 'systemd' | 'darwin' | 'amazon';
 
 type ErrCallback = (err: Error) => void;
-type ErrProcCallback = (err: Error, proc: Proc) => void;
-type ErrProcDescCallback = (err: Error, processDescription: ProcessDescription) => void;
-type ErrProcDescsCallback = (err: Error, processDescriptionList: ProcessDescription[]) => void;
-type ErrResultCallback = (err: Error, result: any) => void;
-type ErrBusCallback = (err: Error, bus: any) => void;
+type ErrProcCallback = (err: Error | null, proc: Proc) => void;
+type ErrProcsCallback = (err: Error | null, procs: Proc[]) => void;
+type ErrProcDescCallback = (err: Error | null, processDescription: ProcessDescription) => void;
+type ErrProcDescsCallback = (err: Error | null, processDescriptionList: ProcessDescription[]) => void;
+type ErrResultCallback = (err: Error | null, result: any) => void;
+type ErrBusCallback = (err: Error | null, bus: any) => void;
