diff --git a/node_modules/pm2-axon/lib/sockets/sock.js b/node_modules/pm2-axon/lib/sockets/sock.js
index 0d01379..ed8156c 100644
--- a/node_modules/pm2-axon/lib/sockets/sock.js
+++ b/node_modules/pm2-axon/lib/sockets/sock.js
@@ -5,7 +5,6 @@
 
 var Emitter = require('events').EventEmitter;
 var Configurable = require('../configurable');
-var debug = require('debug')('axon:sock');
 var Message = require('amp-message');
 var Parser = require('amp').Stream;
 var url = require('url');
@@ -99,7 +98,7 @@ Socket.prototype.pack = function(args){
  */
 
 Socket.prototype.closeSockets = function(){
-  debug('closing %d connections', this.socks.length);
+  console.log('closing %d connections', this.socks.length);
   this.socks.forEach(function(sock){
     sock.destroy();
   });
@@ -116,7 +115,7 @@ Socket.prototype.closeSockets = function(){
  */
 
 Socket.prototype.close = function(fn){
-  debug('closing');
+  console.log('closing');
   this.closing = true;
   this.closeSockets();
   if (this.server) this.closeServer(fn);
@@ -130,7 +129,7 @@ Socket.prototype.close = function(fn){
  */
 
 Socket.prototype.closeServer = function(fn){
-  debug('closing server');
+  console.log('closing server');
   this.server.on('close', this.emit.bind(this, 'close'));
   this.server.close();
   fn && fn();
@@ -160,7 +159,7 @@ Socket.prototype.address = function(){
 Socket.prototype.removeSocket = function(sock){
   var i = this.socks.indexOf(sock);
   if (!~i) return;
-  debug('remove socket %d', i);
+  console.log('remove socket %d', i);
   this.socks.splice(i, 1);
 };
 
@@ -174,7 +173,7 @@ Socket.prototype.removeSocket = function(sock){
 Socket.prototype.addSocket = function(sock){
   var parser = new Parser;
   var i = this.socks.push(sock) - 1;
-  debug('add socket %d', i);
+  console.log('add socket %d', i);
   sock.pipe(parser);
   parser.on('data', this.onmessage(sock));
 };
@@ -195,11 +194,11 @@ Socket.prototype.addSocket = function(sock){
 Socket.prototype.handleErrors = function(sock){
   var self = this;
   sock.on('error', function(err){
-    debug('error %s', err.code || err.message);
+    console.log('error %s', err.code || err.message);
     self.emit('socket error', err);
     self.removeSocket(sock);
     if (!~ignore.indexOf(err.code)) return self.emit('error', err);
-    debug('ignored %s', err.code);
+    console.log('ignored %s', err.code);
     self.emit('ignored error', err);
   });
 };
@@ -275,7 +274,7 @@ Socket.prototype.connect = function(port, host, fn){
     var retry = self.retry || self.get('retry timeout');
     if (retry === 0) return self.emit('close');
     setTimeout(function(){
-      debug('attempting reconnect');
+      console.log('attempting reconnect');
       self.emit('reconnect attempt');
       sock.destroy();
       self.connect(port, host);
@@ -284,7 +283,7 @@ Socket.prototype.connect = function(port, host, fn){
   });
 
   sock.on('connect', function(){
-    debug('connect');
+    console.log('connect');
     self.connected = true;
     self.addSocket(sock);
     self.retry = self.get('retry timeout');
@@ -292,7 +291,7 @@ Socket.prototype.connect = function(port, host, fn){
     fn && fn();
   });
 
-  debug('connect attempt %s:%s', host, port);
+  console.log('connect attempt %s:%s', host, port);
   sock.connect(port, host);
   return this;
 };
@@ -313,12 +312,12 @@ Socket.prototype.onconnect = function(sock){
   else if (sock.server && sock.server._pipeName)
     addr = sock.server._pipeName;
 
-  debug('accept %s', addr);
+  console.log('accept %s', addr);
   this.addSocket(sock);
   this.handleErrors(sock);
   this.emit('connect', sock);
   sock.on('close', function(){
-    debug('disconnect %s', addr);
+    console.log('disconnect %s', addr);
     self.emit('disconnect', sock);
     self.removeSocket(sock);
   });
@@ -368,13 +367,13 @@ Socket.prototype.bind = function(port, host, fn){
 
   this.server = net.createServer(this.onconnect.bind(this));
 
-  debug('bind %s:%s', host, port);
+  console.log('bind %s:%s', host, port);
   this.server.on('listening', this.emit.bind(this, 'bind'));
 
   if (unixSocket) {
     // TODO: move out
-    this.server.on('error', function(e) {
-      debug('Got error while trying to bind', e.stack || e);
+	  this.server.on('error', function (e) {
+		console.log('Got error with code', e.code, 'while trying to bind', e.stack || e);
       if (e.code == 'EADDRINUSE') {
         // Unix file socket and error EADDRINUSE is the case if
         // the file socket exists. We check if other processes
@@ -384,7 +383,7 @@ Socket.prototype.bind = function(port, host, fn){
         var clientSocket = new net.Socket();
 
         clientSocket.on('error', function(e2) {
-          debug('Got sub-error', e2);
+          console.log('Got sub-error', e2);
           if (e2.code == 'ECONNREFUSED' || e2.code == 'ENOENT') {
             // No other server listening, so we can delete stale
             // socket file and reopen server socket
@@ -395,7 +394,8 @@ Socket.prototype.bind = function(port, host, fn){
           }
         });
 
-        clientSocket.connect({path: port}, function() {
+		  clientSocket.connect({ path: port }, function () {
+			console.log('clientSocket connected');
           // Connection is possible, so other server is listening
           // on this file socket
           if (fn) return fn(new Error('Process already listening on socket ' + port));
